assets/less/icons.less

```diff
@@ -29,22 +29,25 @@
 }
 
 .icon-link:before {
-	content: "\e005";
```

Comment:
Why? Actually this is to normalize the indentation. Other `.less` files have 2 spaces for indentation.